app/models/curation_concerns/state_workflow.rb

```diff
@@ -7,7 +7,7 @@ def initialize(state)
     attr_reader :state
 
     def pending?
-      state == :pending
+      state == :pending || state == ::RDF::URI("http://fedora.info/definitions/1/0/access/ObjState#inactive")
```

Comment:
Use `Vocab::FedoraResourceStatus.inactive` instead of initializing a new URI.   Is the `state == :pending` is still relevant after this change?